os/kernel/environ/env_setenv.c

```diff
@@ -195,7 +195,7 @@ int setenv(FAR const char *name, FAR const char *value, int overwrite)
 
 	/* Now, put the new name=value string into the environment buffer */
 
-	sprintf(pvar, "%s=%s", name, value);
+	snprintf(pvar, sizeof(pvar), "%s=%s", name, value);
```

Comment:
Ok. Taken care. Kindly review the latest changes .
https://github.com/Samsung/TizenRT/pull/302/commits/10c7234319115d9bd0b3f413b84864744786eebe